library/core/functions.general.php

```diff
@@ -3110,6 +3110,32 @@ function safeImage($ImageUrl, $MinHeight = 0, $MinWidth = 0) {
     }
 }
 
```

Comment:
isTrustedDomain was added 2 years ago:
https://github.com/vanilla/vanilla/commit/110419b1dfdb747799eac2097e288647d3e891f1
wilcards support was added 10 months ago but we never updated the isTrustedDomain function. (It is used in one place only)
https://github.com/vanilla/vanilla/commit/d2a593f912b0e6debd6f15b7186d571573325043

We should just update isTrustedDomain.
Use http://php.net/manual/en/function.parse-url.php to get the host and pregmatch on that! (preg_quote and replace `\*` by `.*?`) for the preg_match.